subiquity/server/server.py

```diff
@@ -141,7 +141,7 @@ async def client_variant_GET(self) -> str:
         return self.app.variant
 
     async def ssh_info_GET(self) -> Optional[LiveSessionSSHInfo]:
-        ips = []
+        ips: List[str] = []
```

Comment:
It looks like all users of actual_global_ip_addresses want them in strs anyhow, so maybe that should just return them as a list of strs.  Suggest looking into it in a follow-on PR.

Reply:
Yes, that seems correct.
I quite like passing around `ip_address` objects rather than strings though. The API is quite nice and I can imagine that at some point we will have a use-case where we need to do things like the following:

```python
if addr.version == 4:
    print(addr)
else:
    print(f"[{addr}]")
```
We don't have this kind of use-case today so I'm down to change it if you like `str` objects better.

Reply:
Yea, I can see the use case for the objects.  Let's leave it.